apps/test/unit/signUpFlow/FinishStudentAccountTest.tsx

```diff
@@ -1,5 +1,6 @@
 import {render, screen, fireEvent} from '@testing-library/react';
 import React from 'react';
+import sinon from 'sinon'; // eslint-disable-line no-restricted-imports
```

Comment:
Curious why sinon was needed here?

Reply:
Over jest I mean. (I think there's a way to mock with jest.)

Reply:
Though if other files for this project are already using sinon, it makes sense to use sinon here as well!

Reply:
There's a bit more context in [this thread](https://codedotorg.slack.com/archives/C0T0PNR0D/p1727383953287749)- folks have been able to mock with jest but not fetch specifically and not in TS, so I opted to match the other files here to optimize for getting this PR done more quickly.

Reply:
Makes sense!